doc/modules/model_persistence.rst

```diff
@@ -5,22 +5,20 @@ Model persistence
 =================
 
 After training a scikit-learn model, it is desirable to have a way to persist
-the model for future use without having to retrain. The following section gives
-you an example of how to persist a model with pickle. We'll also review a few
-security and maintainability issues when working with pickle serialization.
+the model for future use without having to retrain. The following sections give
+you some hints on how to persist a scikit-learn model.
 
-An alternative to pickling is to export the model to another format using one
-of the model export tools listed under :ref:`related_projects`. Unlike
-pickling, once exported you cannot recover the full Scikit-learn estimator
-object, but you can deploy the model for prediction, usually by using tools
-supporting open model interchange formats such as `ONNX <https://onnx.ai/>`_ or
-`PMML <http://dmg.org/pmml/v4-4/GeneralStructure.html>`_.
+.. note::
+
+   Remember that, once exported in a persistent format, the model could only be
```

Comment:
Uhm I am not sure indeed. With a pickle and if you are lucky, you could do a new round of `partial_fit` to update the model while with ONNX or PMML you will not have access to this method. It was more in this regard that I was thinking to put the note in the section below.

Reply:
I understand... but I don't think it is safe to let people try... I prefer to tell users not to do this anyway: fit is not for persistent storage, if users do it, it is at their own risk.

Reply:
maybe then

could -> should
cannot be -> shouldn't be

so that it's clear that it's a recommendation rather than a technical limitation